.eslintrc.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    extends: ["react-app", "react-app/jest", "eslint:recommended", "plugin:@typescript-eslint/recommended", "prettier"],
+    parser: "@typescript-eslint/parser",
+    plugins: ["@typescript-eslint"],
+    root: true,
+    ignorePatterns: ["src/generated_protos/", "public/", "tailwind.config.js", "build/"],
```

Comment:
I think we can remove `src/generated_protos` and `tailwind.config.js` since those don't exist in this project.